### ISSUE TITLE
Option to prevent overprovisioning drives

### ISSUE THREAD

**Issue by Tokn59** [ALSchwalm/pISO#70]
It seems to be "possible" to create 2 drives which total size is bigger than the physical size of the SD card ?

Kind Regards,

Tony

**Comment by Tokn59** [comment]
Sorry .. just read this in the documentation. It is confusing though ... Maybe in future version option to not vbe able to create oversized/overprovisioned drives ?

**Comment by ALSchwalm** [comment]
I agree that I should probably add an option to prevent a user from doing this, or at least warn when overprovisioning is happening. Thanks for the feedback!
